akd/agents/search/deep_search.py

```diff
@@ -410,147 +353,59 @@ async def _execute_searches(
         queries: List[str],
         original_query: str | None = None,
         is_reformulated: bool = False,
```

Comment:
how do we use `is_reformulated`? Should we add to searchpipeline for this? Curently search pipeline doens't utilize this "is_reformulated" which I presume is needed for Link assessor. Previously, within deep search agent, `reformulated_query` was passed to Link assessor input schema. 

Now in our new refactor, this is set to `None` since there's no param for it in the pipeline input.